application/core/LSYii_Application.php

```diff
@@ -102,6 +102,9 @@ public function __construct($aApplicationConfig = null)
         if (!isset($aApplicationConfig['components']['assetManager']['basePath'])) {
             App()->getAssetManager()->setBasePath($this->config['tempdir'] . '/assets');
         }
+
+        // Load common helper
+        $this->loadHelper("common");
```

Comment:
Probably a good idea, but it would mean we can delete lots of other common_helper includes, right?

Reply:
Hmm.. Honestly, don't recall. This was done long ago.
Let me pick up.

Reply:
It is worth trying... I would just set it as separated DEV task.
We would need to load the helper as well in `application/core/ConsoleApplication.php`.

Maybe we can create a helperLoader class to centralize the loading depending on the application type